### PR TITLE
OCD-3476: Add API AQA tests for surveillance upload

### PR DESCRIPTION
**Pull request by ashwinimore**

### CODE REVIEW

File: chpl/chpl-api/e2e/script/newman-run-dev.js [hunk 1]
```diff
@@ -104,6 +113,7 @@ const jobs = [
     cb => newman.run(statusControllerTests,cb),
     cb => newman.run(testinglabControllerTests,cb),
     cb => newman.run(searchviewControllerTests,cb),
+	cb => newman.run(surveillanceControllerTests,cb),
```
Comment (palakpatel311) [inline chpl/chpl-api/e2e/script/newman-run-dev.js]: Looks like has 2 extra spaces in this line. Same with other newman files.

File: chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,382 @@
...
+					    "rest",
+						"surveillance",
+						"upload"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "POST /surveillance/upload - ROLE_ACB has access to upload surveillance activity on own listings",
```
Comment (palakpatel311) [inline chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json:65]: Can we also add a test mentioned in TC document as "ACB A attempts to add surveillance activity on listing owned ACB B via upload"?
This test is not in UI test so good to add it here.

Comment (ashwinimore) [reply chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json:65]: I'd like to add that test too but I put that one in excluded list due to lack of aqa user account that is attached to one ACB organization only. The one we have now is attached to all active ACB organizations, it won't work for this scenario.

File: chpl/chpl-api/e2e/script/newman-run-dev.js (hunk 1 above)
Comment (ashwinimore) [reply chpl/chpl-api/e2e/script/newman-run-dev.js]: pushed a commit to fix that

File: chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json (hunk 2 above)
Comment (palakpatel311) [reply chpl/chpl-api/e2e/collections/surveillance-controller.postman_collection.json:65]: I see. We will need to request new ACB credentials from Andrew. so we can start having that user for these types of test. Not holding PR for this.

**Review by palakpatel311 — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Pull request merged**